Detections/MultipleDataSources/ExtensiveAWSComputeResurseDeployements.yaml

```diff
@@ -0,0 +1,54 @@
+id: 9e457dc4-81f0-4d25-bc37-a5fa4a17946a
+name: Extensive AWS Compute Resurse Deployements

```

Comment:
Could you also do a cross-cloud version of this where we see suspicious compute deployment in AWS and Azure in same time frame by same actor?

Reply:
We will activate this through correlation rules, grouping multiple IOAIDs and incorporating time difference conditions. This approach will streamline the process and enhance efficiency. However, this isn't necessary at the moment.